recipes/erkir/all/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+cmake_minimum_required(VERSION 3.0)
+
+project(cmake_wrapper)
+
+include(conanbuildinfo.cmake)
+conan_basic_setup()
+
```

Comment:
```suggestion

set(CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS ON)

```

Reply:
This should be done in the library's CMakeLists though, right?

Reply:
I guess the real solution is properly `__declspec(dllexport`-ing library functions. I may be wrong though, don't know anything about Windows tbh.

Reply:
yes, it would be the proper solution, even for GNU compilers (with `__attribute__(visibility("default"))` + hide all symbols), but it should be handled by upstream, a patch would be too big in this recipe I think.